docs/feature_combo.md

```diff
@@ -27,70 +42,255 @@ If you want to add a list, then you'd use something like this:
 ```c
 enum combos {
   AB_ESC,
-  JK_TAB
+  JK_TAB,
+  QW_SFT,
+  SD_LAYER,
 };
 
 const uint16_t PROGMEM ab_combo[] = {KC_A, KC_B, COMBO_END};
 const uint16_t PROGMEM jk_combo[] = {KC_J, KC_K, COMBO_END};
+const uint16_t PROGMEM qw_combo[] = {KC_Q, KC_W, COMBO_END};
+const uint16_t PROGMEM sd_combo[] = {KC_S, KC_D, COMBO_END};
 
 combo_t key_combos[COMBO_COUNT] = {
   [AB_ESC] = COMBO(ab_combo, KC_ESC),
-  [JK_TAB] = COMBO(jk_combo, KC_TAB)
+  [JK_TAB] = COMBO(jk_combo, KC_TAB),
+  [QW_SFT] = COMBO(qw_combo, KC_LSFT)
+  [SD_LAYER] = COMBO(layer_combo, MO(_LAYER)),
 };
 ```
 
 For a more complicated implementation, you can use the `process_combo_event` function to add custom handling.
+Additionally, this example shows how you can leave `COMBO_COUNT` undefined.
 
 ```c
 enum combo_events {
-  ZC_COPY,
-  XV_PASTE
+  EM_EMAIL,
+  BSPC_LSFT_CLEAR,
+  COMBO_LENGTH
 };
+uint16_t COMBO_LEN = COMBO_LENGTH; // remove the COMBO_COUNT define and use this instead!
 
-const uint16_t PROGMEM copy_combo[] = {KC_Z, KC_C, COMBO_END};
-const uint16_t PROGMEM paste_combo[] = {KC_X, KC_V, COMBO_END};
+const uint16_t PROGMEM email_combo[] = {KC_E, KC_M, COMBO_END};
+const uint16_t PROGMEM clear_line_combo[] = {KC_BSPC, KC_LSFT, COMBO_END};
 
-combo_t key_combos[COMBO_COUNT] = {
-  [ZC_COPY] = COMBO_ACTION(copy_combo),
-  [XV_PASTE] = COMBO_ACTION(paste_combo),
+combo_t key_combos[] = {
+  [EM_EMAIL] = COMBO_ACTION(email_combo),
+  [BSPC_LSFT_CLEAR] = COMBO_ACTION(clear_line_combo),
 };
+/* COMBO_ACTION(x) is same as COMBO(x, KC_NO) */
 
 void process_combo_event(uint16_t combo_index, bool pressed) {
   switch(combo_index) {
-    case ZC_COPY:
+    case EM_EMAIL:
       if (pressed) {
-        tap_code16(LCTL(KC_C));
+        SEND_STRING("john.doe@example.com");
       }
       break;
-    case XV_PASTE:
+    case BSPC_LSFT_CLEAR:
       if (pressed) {
-        tap_code16(LCTL(KC_V));
+        tap_code16(KC_END);
+        tap_code16(S(KC_HOME));
+        tap_code16(KC_BSPC);
       }
       break;
   }
 }
 ```
 
-This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V.  But you could change this to do stuff like change layers, play sounds, or change settings.
-
-## Additional Configuration
+This will send "the" if you hit Backspace and T, and clear the current line with Backspace and Left-Shift. But you could change this to do stuff like play sounds or change settings.
```

Comment:
```suggestion
This will send "john.doe@example.com" if you chord E and M together, and clear the current line with Backspace and Left-Shift. You could change this to do stuff like play sounds or change settings.
```